### PR TITLE
Migrate to Junit5 + AssertK

### PR DESCRIPTION
**Pull request by Goooler**
- Native Kotlin supports and more powerful assertions, see https://github.com/willowtreeapps/assertk.
- https://docs.gradle.org/8.9/userguide/performance.html#execute_tests_in_parallel

### CODE REVIEW

**Comment by Goooler** [comment]
May merge #65 first.

**Comment by egorikftp** [comment]
Can be AssertK due to now we don't test any coroutines logic)

**Comment by Goooler** [comment]
You want https://github.com/cashapp/turbine.

**Comment by Goooler** [comment]
Verified, tests are reported at https://scans.gradle.com/s/yz46z7wp4eyzy/tests/overview.

**Comment by egorikftp** [comment]
**turbine** can be added if necessary in future 🙂

File: gradle/libs.versions.toml [hunk 1]
```diff
@@ -9,13 +9,16 @@ koin-compose = "io.insert-koin:koin-compose:1.1.5"
...
 kotlin-stdlib = { module = "org.jetbrains.kotlin:kotlin-stdlib", version.ref = "kotlin" }
-kotlin-test = { module = "org.jetbrains.kotlin:kotlin-test-junit", version.ref = "kotlin" }
 
 tiamat = "io.github.composegears:tiamat:1.1.0-rc02"
 tiamat-koin = "io.github.composegears:tiamat-koin:1.1.0-rc02"
 
 xpp3 = "org.ogce:xpp3:1.1.6"
 
+junit5-jupiter = "org.junit.jupiter:junit-jupiter:5.10.3"
+junit-launcher = { module = "org.junit.platform:junit-platform-launcher" }
```
Comment (Goooler) [inline gradle/libs.versions.toml:19]: Fixed! Check https://scans.gradle.com/s/t2hhis362bmdi/deprecations.

Comment (egorikftp) [reply gradle/libs.versions.toml:19]: Perfect!

**Pull request merged**